lib/analyzer/apt-analyzer.ts

```diff
@@ -3,12 +3,25 @@ import { AnalyzerPkg } from "./types";
 
 export { analyze };
 
-async function analyze(targetImage: string, options?: DockerOptions) {
-  const docker = new Docker(targetImage, options);
-  const dpkgFile = (await docker.catSafe("/var/lib/dpkg/status")).stdout;
+const APT_DPKG_STATUS = "/var/lib/dpkg/status";
+const APT_EXT_STATES = "/var/lib/apt/extended_states";
+
+const APT_PKGFILES = [APT_DPKG_STATUS, APT_EXT_STATES];
+
+export { APT_PKGFILES };
+
+async function analyze(
+  targetImage: string,
+  options?: DockerOptions,
+  files?: {} | void,
+) {
+  if (!files) {
+    files = await new Docker(targetImage, options).extract(APT_PKGFILES);
+  }
+  const dpkgFile = files[APT_DPKG_STATUS] || "";
```

Comment:
What will happen if `parseDpkgFile` receives `""`? I think if `files[APT_DPKG_STATUS]` isn't present, and is needed, we should just return.

Reply:
This is a mandatory file, should we just return or throw some error?